packages/vagov-eslint/.eslintrc

```diff
@@ -81,30 +81,46 @@
     "class-methods-use-this": 0,
     "global-require": 0,
     "lines-around-directive": 0,
+    "no-else-return": 0,
     "no-mixed-operators": 0,
     "no-undef-init": 0,
     "padded-blocks": 0,
     "import/extensions": 0,
     "import/first": 0,
     "import/no-named-as-default": 0,
+    "import/order": 0,
     "import/prefer-default-export": 0,
+    "jsx-a11y/anchor-is-valid": 0,
+    "jsx-a11y/click-events-have-key-events": 0,
     "jsx-a11y/href-no-hash": 0,
+    "jsx-a11y/label-has-associated-control": 0,
+    "jsx-a11y/label-has-for": 0,
```

Comment:
Some of these sound like they'd be useful to have one. Maybe we can make an issue to evaluate them?

Reply:
sounds good- i'll make a ticket for that